recipes/metanetx-post/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "metanetx-post" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 578bcd1080a55ef46c86f37eec1f0bbe5331dc8b583ff982f055e9db9baae781
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --ignore-installed"
+  entry_points:
+    - "mnx-post = metanetx_post.cli.main:cli"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools
+  run:
+    - aioftp
+    - Click
+    - click-log
+    - cobra-component-models
+    - depinfo
+    - httpx
+    - metanetx-sdk
+    - pandas
+    - pydantic
+    - pyparsing
+    - python >=3.8
+    - rdflib
+    - rdkit
+    - SQLAlchemy
```

Comment:
I think this should be `sqlalchemy`

Reply:
The name on PyPI is that https://pypi.org/project/SQLAlchemy/ but they should anyway be normalized so I don't think it's an issue.

Reply:
As a rule we normalize them in the recipe as well.